src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -16,11 +16,51 @@ export const mapBtcTransaction = ({
 	btcAddress: BtcAddress;
 	latestBitcoinBlockHeight: number;
 }): BtcTransactionUi => {
-	const isTypeSend = inputs.some(({ prev_out }) => prev_out.addr === btcAddress);
-	const output: BitcoinOutput | undefined = out.find(({ addr }) =>
-		isTypeSend ? addr !== btcAddress : addr === btcAddress
+	const { totalInputValue, isTypeSend } = inputs.reduce<{
```

Comment:
My understanding is that a transaction is mapped here and if there is no address related to me (neither in `inputs` nor in `outputs`), we still run the loops. So, I just wanted to double confirm: I imagine that you are filtering the transactions before feed them here?

Reply:
We don't need additional filters here since we are fetching transactions for a specific address (endpoint: `{API_URL}/rawaddr/${btcAddress}`). In case there were no transactions related to an address, txs will be an empty array, hence this method won't be called at all.